modules/tinymce/src/themes/mobile/main/ts/util/Thor.ts

```diff
@@ -12,7 +12,7 @@ import { Attr, Css, SelectorFilter } from '@ephox/sugar';
 const attr = 'data-ephox-mobile-fullscreen-style';
 const siblingStyles = 'display:none!important;';
 const ancestorPosition = 'position:absolute!important;';
-/// TINY-3407 ancestors need 'height:100%!important;overflow:visible!important;' to prevent collapsed ancestors hiding the editor
+// / TINY-3407 ancestors need 'height:100%!important;overflow:visible!important;' to prevent collapsed ancestors hiding the editor
```

Comment:
Does `///` have special meaning? If so then this could be a problem.

Reply:
not that i know of, appears to be a left over typo

Reply:
There were a few scattered through the code. I've removed the extra slash.